ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml

```diff
@@ -179,6 +179,23 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-additional-ca-policy
+- as: aws-ipi-custom-sg-default-machine-arm-f28
```

Comment:
Per https://github.com/openshift/release/pull/58256#discussion_r1823823508, we can update the following jobs, instead of adding new jobs:

`periodic-ci-openshift-openshift-tests-private-release-4.17-multi-nightly-aws-ipi-usertags-custom-sg-fips-amd-f28-destructive`
`periodic-ci-openshift-openshift-tests-private-release-4.18-multi-nightly-aws-ipi-usertags-custom-sg-fips-amd-f28-destructive`

Reply:
OK, that's a good idea.

Reply:
Updated